src/v1/types/task.rs

```diff
@@ -153,15 +164,19 @@ pub struct TaskLog {
     pub logs: Vec<executor::Log>,
 
     /// The start time.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     pub start_time: Option<DateTime<Utc>>,
 
     /// The end time.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     pub end_time: Option<DateTime<Utc>>,
 
     /// The output file logs.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
```

Comment:
This is a required key, so I'm going to remove this.

Reply:
good catch